### PR TITLE
Dev#97

### PR DESCRIPTION
**Pull request by shahules786**
PR to implement crop transform https://github.com/asteroid-team/torch-audiomentations/issues/97

### CODE REVIEW

**Comment by iver56** [comment]
I believe a common use case for this transform will be to adapt the audio to the constant length requirement of a ML model. Such length requirements can be oddly specific, for example 461906 samples. I would like to have an option to specify the desired output length in terms of samples. Number of seconds also makes sense in some cases :)

Maybe you could add support for specifying the length in samples, just like in `Shift`?

https://github.com/asteroid-team/torch-audiomentations/blob/master/torch_audiomentations/augmentations/shift.py#L69

**Comment by shahules786** [comment]
Yes, I have added the argument @iver56 .

File: torch_audiomentations/augmentations/random_crop.py [hunk 1]
```diff
@@ -7,14 +7,30 @@
...
+        """
         super(RandomCrop, self).__init__()
         self.sampling_rate = sampling_rate
-        self.num_samples = int(self.sampling_rate * seconds)
+        if max_length_unit == "seconds":
+            self.num_samples = int(self.sampling_rate * max_length)
+        elif max_length_unit == "samples":
+            self.num_samples = int(max_length)
+        else:
+            raise ValueError('max_length must be "samples" or "seconds"')
```
Comment (iver56) [inline torch_audiomentations/augmentations/random_crop.py:33]:
```suggestion
            raise ValueError('max_length_unit must be "samples" or "seconds"')
```

Comment (shahules786) [reply torch_audiomentations/augmentations/random_crop.py]: Thanks @iver56 , done.

**Comment by iver56** [comment]
Thanks! I'm not sure if I can squeeze this into the upcoming release, as there will also be a lot going on with https://github.com/asteroid-team/torch-audiomentations/pull/123 that changes the API of things

But I can at least merge it now, and then maybe massage it (like add target support) and add it to the API later 👍

**Pull request merged**